README.md

```diff
@@ -1,123 +1,15 @@
-# Differ.rs
-[![License](https://img.shields.io/badge/license-MIT%20%26%20Apache%202.0-green)](#license)
-[![CI](https://github.com/nlp-rs/differ.rs/actions/workflows/main.yml/badge.svg)](https://github.com/nlp-rs/differ.rs/actions/workflows/main.yml)
-[![Security audit](https://github.com/nlp-rs/differ.rs/actions/workflows/security-audit.yml/badge.svg)](https://github.com/nlp-rs/differ.rs/actions/workflows/security-audit.yml)
-> warning: **Differ.rs is currently experimental**
-This crate provides edit distance, delta vectors between 2 words, and lets you apply delta vectors in order to transform words.
+# differ.rs
 
-## Install
-```shell
-cargo add differ-rs
-```
-or, simply add the following string to your Cargo.toml:
+Differ.rs is a collection of natural langauge processing crates, all written in Rust.
 
-```toml
-differ-rs = "0.0.0"
-```
+# *Brief overview*
+### Crates
+ - [x] [`differ`](./crates/differ): Provides edit distance, delta vectors between 2 words, and word transformation 
+   - [x]  `apply_diff`: Allows users to apply delta vectors in order to transform a words.
+   - [x]  `extra_traits`: all `struct`s implemented in `differ-rs` are `HammingDistance` and `LevenshteinDistance`. Each Struct implements the `diff` and `distance` methods.
+ - [ ] `bio_dif`: Proivdes functions in order to to align protein or nucleotide sequences.
```

Comment:
A few typos: `bio_diff`, `Provides`. This sentence can also be more succinct: "Allows aligning protein and nucleotide sequences`